documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -178,12 +181,16 @@ Contains 32-bit Unix epoch of the date and time in UTC the record was written to
 | 24 | 2 | | Event type
 | 26 | 2 | | Number of strings
 | 28 | 2 | | Event category
-| 30 | 2 | | [yellow-background]*Unknown (Event flags)* +
-Actual usage unknown indicated as reserved
-| 32 | 4 | | [yellow-background]*Unknown (Closing record number)* +
-Actual usage unknown indicated as reserved
+| 30 | 2 | | *ReservedFlags (Event flags)* +
+Always 0 +
+`[MS-EVEN]` indicates the value 0x8000 has meaning, but
+no version of the Event Log uses this flag for any purpose
+| 32 | 4 | | *Reserved (Closing record number)* +
+Always 0
```

Comment:
reword to "MS-EVEN states that this value should always 0"

I've seen multiple times that MS documentations claims one thing but in practice the value is not 0 due to optimization.